test/bin/ci_phase_test.sh

```diff
@@ -14,6 +14,8 @@ if [ ! -d "${RF_VENV}" ]; then
     "${ROOTDIR}/scripts/fetch_tools.sh" robotframework
 fi
 
+bash -x ./bin/manage_vm_connections.sh local
```

Comment:
This is no longer necessary. The scenario script does the same thing as it builds each VM.